BrainPortal/app/views/groups/new.html.erb

```diff
@@ -54,6 +54,12 @@
       <%= f.check_box :track_usage %>
     <% end %>
 
+    <% Make project public, so its files are visible to all the users %>
```

Comment:
I've not looked at how the form is rendered, but is this layout correct? In the original form, we had the explanations before the inputs, and you added your explanation after your input.

Reply:
corrected